tests/std/tests/P0896R4_ranges_alg_move_backward/test.cpp

```diff
@@ -88,7 +91,52 @@ struct instantiator {
     }
 };
 
+constexpr void test_memmove() {
+    // Get some coverage for the memmove optimization, which we would not otherwise have since we do not currently
+    // unwrap output iterators. TRANSITION, GH-893
+    using ranges::move_backward, ranges::move_backward_result, ranges::begin, ranges::end, ranges::equal;
+
+    struct S { // move-only and trivially copyable
+        int val = 10;
+
+        constexpr S() = default;
+        constexpr S(int x) : val{x} {}
+        constexpr S(S&&)     = default;
+        constexpr S& operator=(S&&)      = default;
+        auto operator<=>(const S&) const = default;
+    };
+
+
+    { // Validate range overload
+        S input[]                                               = {13, 42, 1729};
+        S output[]                                              = {-2, -2, -2};
+        const same_as<move_backward_result<S*, S*>> auto result = move_backward(input, end(output));
+        assert(result.in == end(input));
+        assert(result.out == begin(output));
+        assert(equal(output, input));
+    }
+    { // Validate iterator + sentinel overload
+        S input[]                                               = {13, 42, 1729};
+        S output[]                                              = {-2, -2, -2};
+        const same_as<move_backward_result<S*, S*>> auto result = move_backward(begin(input), end(input), end(output));
+        assert(result.in == end(input));
+        assert(result.out == begin(output));
+        assert(equal(output, input));
+    }
+    { // Validate overlapping ranges
+        S io[]                                                  = {0, 1, 2, 42};
+        const same_as<move_backward_result<S*, S*>> auto result = move_backward(io + 0, io + 3, io + 4);
+        assert(result.in == io + 3);
+        assert(result.out == io + 1);
+        const int expected[] = {0, 0, 1, 2};

```

Comment:
Could be `static constexpr`.

Reply:
It could be `constexpr`, but not `static`: `constexpr` functions cannot define variables with `static` and `thread` storage duration (this is why so many of the Ranges tests define `static constexpr` class members despite that they are only used in a single member function).
```suggestion
        constexpr int expected[] = {0, 0, 1, 2};
```